django_celery_tracker/decorators.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import redirect
+from django.urls import reverse
+
+
+# Similar to user_passes_test but redirects to django admin login
+def admin_required(func):
+    def as_view(request, *args, **kwargs):
+        if not request.user.is_superuser:
+            return redirect(
```

Comment:
Should this log the user out before doing the redirect?

Reply:
In my testing, no. The django admin login view does the logout for you.